### ISSUE TITLE
no way to turn off tar output

### ISSUE THREAD

**Issue by dericed** [NMAAHC/nmaahcmm#82]
I found that in the current version of camera_cards, there seems to be no way to turn off the tar output. There's two flags to handle the outputs. 

```
  -t tar camera files and folders into a tarball, compressed with gzip
  -a create packaged AIP; this is the default option, but you can use this flag to specify you want an AIP alongside a tarball (e.g. '-ta')
```

But regardless, the tar is output. Should I change it so there's no tar output unless `-t` is set, and no aip output unless `-a` is set? Or we could change syntax to something like `-t y -a n` (for yes to tar and no to aip)?

Nudge @chialinchou1

**Comment by chialinchou1** [comment]
Thanks @dericed 

Should we just make `-a ` creating an AIP without concatenated the video files (just sort files) and `-ta` to tar the result AIP into a compressed tallball? 

For the default option (no flag), would we ever need a tarball for the AIP with concatenated video files? @BleakleyMcD

**Comment by BleakleyMcD** [comment]
@dericed I don't think we are getting tar balls unless we use the -t flag... @chialinchou1 confirm?

based on what Chialin is suggesting, how about:
-a = make the aip, don't concatenate the files
-c = concatenate the files, don't make the aip
-t = make a tar of files and folders, don't do anything else

so, we could have combos like -ac or -ct or -at

I think -ac is probably the most commonly used

**Comment by BleakleyMcD** [comment]
@dericed @chialinchou1 hi :)
